src/vellum/workflows/nodes/displayable/inline_prompt_node/node.py

```diff
@@ -48,26 +58,17 @@ def run(self) -> Iterator[BaseOutput]:
         string_outputs = []
         json_output = None
 
-        should_parse_json = False
-        if hasattr(self, "parameters"):
-            custom_params = self.parameters.custom_parameters
-            if custom_params and isinstance(custom_params, dict):
-                json_schema = custom_params.get("json_schema", {})
-                if (isinstance(json_schema, dict) and "schema" in json_schema) or custom_params.get("json_mode", {}):
-                    should_parse_json = True
```

Comment:
Whatever existing test we have on this, let's update to use `parameterize()` on json_mode being present and not present and this still working